nav2_costmap_2d/include/nav2_costmap_2d/testing_helper.hpp

```diff
@@ -1,3 +1,40 @@
+/*********************************************************************
```

Comment:
Are we sure about the origin of this source? Just wondering why the files didn't have the  Willow Garage headers in the first place?

Reply:
Not 100%, but the git blame seems to be the same originators as the other code in the package

Reply:
I think I'd just disable this linter check, rather than assign a copyright when we can't really tell the origin. Git blame can be misleading. It just shows the last person to touch that area of code. That's probably why cpplint and copyright were disabled in the CMakeLists.txt

Reply:
Is there a way to disable just the copyright check in the cpplint? Or would it be fine to just disable cpplint?